src/components/shared/formItems/BondingCurve/ProjectBondingCurveRate.tsx

```diff
@@ -112,13 +117,21 @@ export default function ProjectBondingCurveRate({
       label={
         hideLabel ? undefined : (
           <div>
-            <Trans>Bonding curve rate</Trans>{' '}
+            <span>{label ? label : t`Bonding curve rate`}</span>{' '}
```

Comment:
we can basically avoid `t` in JSX altogether for the most part. no big deal though